scripts/python/verify_agent_os.py

```diff
@@ -0,0 +1,14 @@
+# Script verifies the operating system for the platform on which it is being run
+# Used in build pipelines to verify the build agent os
+# Variable: The name of the os to verfy against
+
+import sys
+import platform
+osparameter = sys.argv[1]
```

Comment:
This is a nit, but also something that I really do like across the board. For variable naming, can we use `snake_case` please?